pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[project]
+name = "trnpy"
+version = "0.0.1"
+authors = [
+  { name="John Dyreby", email="john@isentropic.dev" },
+  { name="Greg Troszak", email="greg@isentropic.dev" },
+]
+description = "A Python wrapper for TRNSYS"
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+]
```

Comment:
I have no idea if these are appropriate, but that's what the tutorial had it seems like a sane starting point.